src/WebJobs.Extensions.DurableTask.Analyzers/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Activity function call specifies a parameter that doesn't match the function definition parameter.</value>
   </data>
   <data name="ActivityArgumentAnalyzerMessageFormat" xml:space="preserve">
-    <value>Activity function named '{0}' takes a parameter of type '{1}' but was given '{2}'.</value>
+    <value>Activity function named '{0}' takes a parameter of type '{1}' but was given '{2}'</value>
```

Comment:
How do we determine when to end with a period and when not to? It seems like we're inconsistent here. For example, line 121 and 136 end with a period.

Reply:
This is also type hinted by the analyzers for the Roslyn SDK. Long story short, Title and MessageFormat strings should not end with punctuation unless they are multi-sentence strings. Description strings always have punctuation. Strange, but that's the guidance